definitions/region/README.md

```diff
@@ -142,6 +142,18 @@ Example:
 
 > Norway>Germany
 
+To represent data that refers to a flow or cost contribution of a generator
```

Comment:
```suggestion
To represent data that refers to a flow or cost allocated to a generator
```

Reply:
If I understood well, there is an allocation of responsibilities in the flow of a particular transmission line here. Then, the word cost allocated is more precise.